src/main/java/build/buildfarm/cas/cfc/CASFileCache.java

```diff
@@ -325,6 +333,15 @@ public CASFileCache(
     this.delegateSkipLoad = delegateSkipLoad;
     this.directoriesIndexDbName = directoriesIndexDbName;
 
+    if (publishTtlMetric) {
+      casTtl =
+          Histogram.build()
```

Comment:
Do default buckets work for this case? I think we had to make some adjustments for some of the metrics to make them useful.

Reply:
Oh good point.  I'm currently graphing it, and grafana's histogram made it look like we weren't keep cas for very long.  Changing the metric to seconds now and adding bucket sizes 1hr through 2 weeks.